### PR TITLE
WD for Proximity Sensor spec. 

### PR DESCRIPTION
**Pull request by riju**
Create a release for the WD draft of the generic-sensor based Proximity Sensor specification.

### CODE REVIEW

File: releases/WD-2016-07-14.html [hunk 1]
```diff
@@ -0,0 +1,657 @@
+<!doctype html><html lang="en">
```
Comment (marcoscaceres) [inline releases/WD-2016-07-14.html:1]: I'm not sure why this file is needed at all? These documents are all on the W3C server. Additionally, seems redundant to include them in a release/ folder. If anything, just `git tag` the appropriate release with a date. It's much more useful than having redundant generated files in a release folder.

File: index.bs [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <pre class="metadata">
 Title: Proximity Sensor
 Level: 1
-Status: ED
+Status: WD
 ED: https://w3c.github.io/proximity/
+TR: https://www.w3.org/TR/proximity/releases/WD-2016-07-12.html
```
Comment (marcoscaceres) [inline index.bs]: Same here. Just change to WD and auto-release that instead of a snapshot.

File: releases/WD-2016-07-14.html (hunk 1 above)
Comment (anssiko) [inline releases/WD-2016-07-14.html:1]: AFAIK, Bikeshed does not yet co-operate with Echidna and Travis, see: https://github.com/w3c/sensors/issues/74#issuecomment-203350830

(Ping @dontcallmedom for the latest status.)

We can remove the staged snapshot from `releases/` folder when the spec gets published on TR, and tag the right commit. It is not unusual one needs to fine-tune the boilerplate, publish date etc. so staging a snapshot has proven to be practical (could do it in its own branch as well).

Comment (marcoscaceres) [inline releases/WD-2016-07-14.html:1]: @anssiko, apologies for the noise. I had assumed bs would have been working 😳

**Comment by anssiko** [comment]
Fixes some bugs in another PR #16, closing this one.

**Pull request closed without merging**